rsc/src/main/scala/rsc/scan/Scanner.scala

```diff
@@ -261,8 +261,9 @@ final class Scanner private (
       nextChar()
```

Comment:
The commit message has a typo.

Reply:
Last minute dash as my train pulled in - will fix.

Reply:
The commit message does not include the benchmark info - I can add it if desired.

Reply:
Thanks for the suggestion! Let me finish with `bin/benchQuick`, and then we'll rerun the benchmarks.

Reply:
Almost there infrastructure-wise. Please take a look at https://github.com/twitter/reasonable-scala/pull/38.

Reply:
Also, there's still a typo in the commit message :) I'd expect it to say "Avoid repeated calls to lexeme in hexadecimalNumber".

Reply:
Do'h!   I stared at the commit message after your last comment and still missed it....

Reply:
fixed now....